js/app/modules/arrangement/sideBySide.js

```diff
@@ -3,40 +3,61 @@
 /* exported SideBySide */
 
 const Endless = imports.gi.Endless;
+const Gdk = imports.gi.Gdk;
 const Gtk = imports.gi.Gtk;
 
 const Arrangement = imports.app.interfaces.arrangement;
 const Knowledge = imports.app.knowledge;
 const Module = imports.app.interfaces.module;
+const Utils = imports.app.utils;
 
 const MENU_HEIGHT = 50;
-const _HorizontalSpacing = {
-    TINY: 15,
-    SMALL: 20,
-    LARGE: 40,
-    XLARGE: 50,
-};
+const SPACING_MIN = 15;
+const SPACING_MAX = 50;
 
 const _SideBySideLayout = new Knowledge.Class({
     Name: 'Arrangement.SideBySideLayout',
-    Extends: Endless.CustomContainer,
+    Extends: Gtk.Box,
 
     _init: function (props={}) {
         this.all_visible = true;
+        props.orientation = Gtk.Orientation.HORIZONTAL;
 
         this.parent(props);
-    },
 
-    // Removing a visible widget should recalculate the positions of all widgets
-    vfunc_remove: function (widget) {
-        let needs_resize = widget.get_child_visible();
-        this.parent(widget);
-        if (needs_resize)
-            this.queue_resize();
-    },
+        /* Button to trigger dropdown */
+        this._popover_button = new Gtk.Button({ label: " ⁝ " });
+        Utils.set_hand_cursor_on_widget(this._popover_button);
+        this.pack_end(this._popover_button, false, false, 0);
+
+        let label = this._popover_button.get_child();
+        label.valign = Gtk.Align.CENTER;
+
+        /* Used for dropdown menu if cards do not fix */
+        this._popover = new Gtk.Popover({
+            position: Gtk.PositionType.BOTTOM,
+            relative_to: label,
+        });
+        this._popover_button.connect("clicked", () => {
+            this._popover.popup();
+        });
+
+        /* Classes needed for cards to have the same style */
+        let context = this._popover.get_style_context();
+        context.add_class('ArrangementSideBySide__popover');
+        context.add_class('ArrangementSideBySide');
+        context.add_class('Arrangement');
 
-    vfunc_get_request_mode: function () {
-        return Gtk.SizeRequestMode.CONSTANT_SIZE;
+        /* Dropdown contents */
+        let scroll = new Gtk.ScrolledWindow ({
+            vscrollbar_policy: Gtk.PolicyType.NEVER,
```

Comment:
Why never? Agreed it's unlikely we'll have more overflow categories than can fit on the screen though